sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -36,29 +36,41 @@ import org.apache.spark.sql.internal.StaticSQLConf._
 import org.apache.spark.status.ElementTrackingStore
 import org.apache.spark.util.{MutableURLClassLoader, Utils}
 
-
```

Comment:
Let's get rid of newline changes. It's going to make backporting and reverting harder potentially.